### PR TITLE
All test cases are passing

### PR DESCRIPTION
**Pull request by vitalyops**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -31,16 +31,28 @@ def reservations_get():
         resp = Response(js, status=200, mimetype='application/json')
         return resp
 
+# Validate JSON data
+def is_json(myjson):
```
Comment (nilgradisnik) [inline server.py:35]: Well done 👍 That's a good way to test if this is valid JSON

File: server.py [hunk 2]
```diff
@@ -31,16 +31,28 @@ def reservations_get():
...
+  return True
+
 @app.route('/reservations', methods = ['POST'])
 def reservations_post():
     payload = request.get_json()
-    print(payload['hour'])
-    if not request.is_json:
+    print(payload)
+    if is_json(request) is False:
+        print(payload)
```
Comment (nilgradisnik) [inline server.py:47]: You can remove these `print` lines, I'm guessing you added that to debug

**Review by nilgradisnik — APPROVED** [review]
Tests are passing 👏

File: server.py (hunk 2 above)
Comment (vitalyops) [reply server.py:47]: Oops. Thanks.

**Pull request merged**